gordo_components/model/factories/lstm_autoencoder.py

```diff
@@ -117,8 +111,8 @@ def lstm_symmetric(
     n_features: int,
     n_features_out: int = None,
     lookback_window: int = 1,
-    dims: Tuple[int, ...] = (256, 128, 64),
-    funcs: Tuple[str, ...] = ("tanh", "tanh", "tanh"),
+    dims: Tuple[int, ...] = (),
+    funcs: Tuple[str, ...] = (),
```

Comment:
Should have the default values here. 
Right now, all of our `Keras..` models can be initialized and _"just work"_ by supplying the `kind` parameter (and we'll eventually have a default for that as well, to be more scikit-learn players) 

However, removing defaults here will cause the construction of the model to fail because of the first check in this very function.

Reply:
Thanks Miles, I missed this one. Strange the tests passed... I'll take a look there and see if I can add something that would make that fail next time.